styles/style.css

```diff
@@ -0,0 +1,1681 @@
+  padding: 0.5rem 1rem;
+  text-decoration: none;
+}
+
+.btn:hover {
+  cursor: pointer;
+}
+
+.btn:focus {
+  outline: thin dotted;
+}
+
+.card {
+  max-width: 60%;
+  background-color: #F9FAFB;
+  -webkit-box-shadow: 1px 1px 5px 1px rgba(0, 0, 0, 0.1);
+          box-shadow: 1px 1px 5px 1px rgba(0, 0, 0, 0.1);
+  border-radius: 0.5rem;
+  padding: 0.5rem;
+  -ms-flex-item-align: center;
+      -ms-grid-row-align: center;
+      align-self: center;
+}
+
+@media only screen and (max-width: 750px) {
+  .card {
+    max-width: 80%;
+  }
+}
+
+@media only screen and (max-width: 500px) {
+  .card {
+    max-width: 90%;
+  }
+}
+
+@media only screen and (max-width: 390px) {
+  .card {
+    max-width: 99%;
+  }
+}
+
+.card--col {
+  max-width: 33%;
+  max-height: 45%;
+  padding: 0;
+}
+
+@media only screen and (max-width: 750px) {
+  .card--col {
+    max-width: 40%;
+    max-height: 45%;
+  }
+}
+
+@media only screen and (max-width: 500px) {
+  .card--col {
+    max-width: 55%;
+  }
+}
+
+@media only screen and (max-width: 390px) {
+  .card--col {
+    max-width: 67%;
+  }
+}
+
+@media only screen and (max-width: 345px) {
+  .card--col {
+    max-width: 80%;
+  }
+}
+
+.card__image {
+  max-width: 100%;
+  max-height: 100%;
+  border-radius: 0.5rem;
+  -o-object-fit: contain;
+     object-fit: contain;
+}
+
+.card__header {
+  max-width: 35%;
+}
+
+.card__header--col {
+  max-width: 100%;
+  max-height: 50%;
+  -ms-flex-item-align: center;
+      -ms-grid-row-align: center;
+      align-self: center;
+}
+
+.card__content {
+  max-width: 63%;
+}
+
+.card__content--col {
+  padding: 0 0.7rem 0.2rem;
+  max-width: 100%;
+  min-height: 50%;
+}
+
+.card__icon {
+  cursor: pointer;
+  font-size: 1.3em;
+  margin-right: 0.5rem;
+}
+
+.card__icon:hover {
+  color: var(--secondary-color);
+}
+
+.card__icon--col {
+  position: absolute;
+  right: 0.5vw;
+  top: 3vh;
+}
+
+.card__strike {
+  text-decoration: line-through;
+  font-size: 0.75em;
+}
+
+.image {
+  max-width: 25%;
+  -ms-flex-item-align: center;
+      -ms-grid-row-align: center;
+      align-self: center;
+}
+
+@media only screen and (max-width: 500px) {
+  .image {
+    max-width: 33%;
+  }
+}
+
+@media only screen and (max-width: 390px) {
+  .image {
+    max-width: 45%;
+  }
+}
+
+@media only screen and (max-width: 390px) {
+  .image {
+    max-width: 50%;
+  }
+}
+
+.image--rounded {
+  border-radius: 50%;
+}
+
+.input {
+  padding: 0.3rem 0.2rem;
+  border: 1px solid var(--primary-color);
+  outline: 1px solid var(--primary-color);
+  color: var(--primary-color);
+}
+
+.input-label {
+  margin: 0.5rem 0;
+  color: var(--primary-color);
+}
+
+.input-required {
+  margin-top: 0.5rem;
+}
+
+.input-container {
+  max-width: 50%;
+}
+
+@media only screen and (max-width: 390px) {
+  .input-container {
+    max-width: 60%;
+  }
+}
+
+@media only screen and (max-width: 345px) {
+  .input-container {
+    max-width: 70%;
+  }
+}
+
+.input--error {
+  border-color: #DC2626;
+  outline-color: #EF4444;
```

Comment:
since variables are used for colors everywhere, it would be nice to include these too in variables